texar/torch/data/data/record_data.py

```diff
@@ -256,11 +324,11 @@ class RecordData(DatasetBase[Dict[str, Any], Dict[str, Any]]):
             hparams={
                 'dataset': {
                     'files': 'image1.pkl',
-                    'feature_original_types': {
-                        'height': ['int64', 'FixedLenFeature'],
-                        'width': ['int64', 'FixedLenFeature'],
-                        'label': ['int64', 'FixedLenFeature'],
-                        'image_raw': ['string', 'FixedLenFeature']
+                    'feature_types': {
+                        'height': ['int64', 'list'],
+                        'width': ['int64', 'list'],
+                        'label': ['int64', 'stacked_tensor'],
```

Comment:
Why `height`/`width` are `list` but `label` is `stacked_tensor`? They are all single integers

Reply:
This is pretty much an arbitrary choice. I chose so because `label` will probably used in the loss function which requires inputs to be tensors, while `height` and `width` are probably not going to be used this way. If this causes confusion then maybe I can change everything to `'stacked_tensor'`.

Reply:
We can probably just add a comment to `height`/`width` saying "`stacked_tensor` would also be used here"

Reply:
Addressed in the new commit.